run.sh

```diff
@@ -187,22 +187,26 @@ check_database() {
 
     wp db create |& logger
 
+    wp core install |& logger
```

Comment:
Install wordpress no matter what, even if importing databases (just to ensure we have all the tables if doing a partial import).